tests/StrictQuoteLineTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace ClickHouseDB\Tests;
+
+use PHPUnit\Framework\TestCase;
+use ClickHouseDB\Quote\StrictQuoteLine;
+
+/**
+ * Class StrictQuoteLineTest
+ * @package ClickHouseDB\Tests
+ */
+class StrictQuoteLineTest extends TestCase
+{
+    use WithClient;
+
+    public function setUp()
+    {
+        $this->client->write('DROP TABLE IF EXISTS cities');
+        return $this->client->write('
+            CREATE TABLE IF NOT EXISTS cities (
+                date Date,
+                city String,
+                keywords Array(String),
+                nums Array(UInt8)
+            ) ENGINE = MergeTree(date, (date), 8192)
+        ');
+    }
+
+    /**
+     * @group test
+     */
+    public function testQuoteValueCSV()
+    {
+        $strict = new StrictQuoteLine('CSV');
+
+        $rows = [
+            ['2018-04-01', '"That work"', ['\"That does not\"', 'That work'], [8, 7]],
+            ['2018-04-02', 'That work', ['\""That does not\""', '"\'\""That work"""\"'], [1, 0]],
+            ['2018-04-03', 'That work', ['\"\"That does not"\'""', '""""That work""""'], [9, 121]],
+        ];
+        $fileName = $this->tmpPath . '__test_quote_value.csv';
+
+        @unlink($fileName);
+        foreach ($rows as $row) {
+            file_put_contents($fileName, $strict->quoteRow($row) . "\n", FILE_APPEND);
+        }
+
+        $this->client->insertBatchFiles('cities', [$fileName], ['date', 'city', 'keywords', 'nums']);
+        $statement = $this->client->select('SELECT * FROM cities');
+
+        $result = array_map('array_values', $statement->rows());
+        foreach ($result as $key => $value) {
+            // check correct quote string
+            $this->assertEmpty(array_diff($rows[$key], $value));
+            $this->assertEmpty(array_diff($rows[$key][2], $value[2]));
+        }
+
+        $rows[0][2][1] = 'Not the same string';
+        $this->assertCount(1, array_diff($rows[0][2], $result[0][2]));
+    }
+}
```

Comment:
missing trailing space, simply add a line break here

probably CI check will discover it after tests pass